src/components/taskItem/TaskItem.tsx

```diff
@@ -1,36 +1,30 @@
 import styles from "./TaskItem.module.css";
 import { ReactElement } from "react";
-import { Task, Status } from "../../types/Task.types";
-import { deleteTask, getTasks, updateTask } from "../../utility/api";
+import { Task, TaskStatus } from "../../types/Task.types";
+import { useAppDispatch } from "../../hooks/reduxHooks";
+import { completeTask, deleteTask } from "../../store/features/tasksSlice";
 
-type Props = Task & {
-  setTasks: (tasks: ReadonlyArray<Task>) => void;
-};
+type Props = Task;
```

Comment:
This is not necessary, since you're not adding anything to the type Task or creating a new type, you can ditch it and use directly the type Task in the type definition of the prams.
```
export const TaskItem = ({ id, title }: Task): ReactElement
```

Reply:
It works, thank you! Just updated :D